src/chrome/content/rules/ilcdn.fi.xml

```diff
@@ -1,5 +1,7 @@
 <ruleset name="ilcdn.fi">
         <target host="static.ilcdn.fi" />
```

Comment:
Move this to a comment above the header, such as:

```
<!--
	Active mixed content:
		- static.ilcdn.fi
-->
```

Reply:
Done. https://github.com/EFForg/https-everywhere/pull/15266/commits/700e0ffb6135b719741147cacb958e79a4201b28